src/root.component.tsx

```diff
@@ -37,6 +37,23 @@ defineConfigSchema("@pih/esm-referrals-queue", {
           "/pihcore/visit/visit.page?patient=${patientUuid}&visit=${visitUuid}&suppressActions=true#/overview",
         validators: [validators.isString, validateUrlTemplate]
       }
+    },
+    homeVisitForm: {
+      spa: {
+        default: false,
+        validators: [validators.isBoolean]
+      },
+      url: {
+        default:
+          "/htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId=${patientUuid}&visitId=${visitUuid}&encounterId=${encounterUuid}&definitionUiResource=file:configuration/pih/htmlforms/section-mch-referral.xml",
+        validators: [validators.isString, validateUrlTemplate]
+      }
+    }
+  },
+  pendingStatuses: {
+    default: ["Pending status", "Referral unmet"],
```

Comment:
We are likely going to have to deal with localizing these?  I forget if the report returns localized statuses or not, could you check when you get a chance?

Reply:
Yes, that's the case. Maybe when it's time to do that is when I'll introduce Dave to the config file, and he can start managing some of these things.